src/wordpiece_tokenizer.cpp

```diff
@@ -94,19 +101,40 @@ bool WordpieceTokenizer::evaluate(ov::TensorVector& outputs, const ov::TensorVec
         new_begins[seq] = ragged_offset;
 
         for(size_t ragged_col = ragged_begins[seq]; ragged_col < ragged_ends[seq]; ++ragged_col) {
-
-            auto str = std::string(chars + begins[ragged_col], chars + ends[ragged_col]);
-            std::vector<core::Token> results = m_tokenizer->Tokenize(str);
-
-            for (const core::Token& token : results) {
+            
+            auto text_orig = std::string(chars + begins[ragged_col], chars + ends[ragged_col]);
+            auto text = text_orig;
+
+            auto text_vec = std::vector<unsigned char>(text.begin(), text.end());
```

Comment:
We can avoid the intermediate `std::string` creation entirely, and make `std::vector` directly from the tensor.

Reply:
also thought about that. I'll improve that

Reply:
agreed, moreover used here string_view right away